src/foundation.cs

```diff
@@ -11719,9 +11721,9 @@ interface NSPortDelegate {
 	}
 
 	[BaseType (typeof (NSObject))]
-	[NoMacCatalyst]
+	[MacCatalyst(15, 0)]
 	interface NSPortMessage {
-#if MONOMAC
+#if MONOMAC || __MACCATALYST__
```

Comment:
same

Reply:
This one explicitly not done, as noted below.